### PR TITLE
chore: setup project and deploy to vercel

### PR DESCRIPTION
**Pull request by devjiwonchoi**
- prettier & run lint-fix
- node & pnpm version
- connect vercel

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -54,9 +58,14 @@
       "last 1 safari version"
     ]
   },
-  "devDependencies": {
-    "@types/react-router-dom": "^5.3.3",
-    "@types/react-slick": "^0.23.10",
-    "@types/react-transition-group": "^4.4.10"
-  }
+  "prettier": {
```
Comment (devjiwonchoi) [inline package.json:61]: setup prettier

File: package.json [hunk 2]
```diff
@@ -54,9 +58,14 @@
...
+    "semi": false,
+    "singleQuote": true,
+    "jsxSingleQuote": false,
+    "trailingComma": "es5"
+  },
+  "engines": {
+    "node": ">=18.17.0"
```
Comment (devjiwonchoi) [inline package.json:68]: nodejs version

File: package.json [hunk 3]
```diff
@@ -54,9 +58,14 @@
...
+  },
+  "packageManager": "pnpm@8.15.4"
```
Comment (devjiwonchoi) [inline package.json:70]: pnpm version

File: package.json [hunk 4]
```diff
@@ -1,40 +1,44 @@
...
-    "typescript": "^4.9.5",
-    "web-vitals": "^3.1.1"
+    "slick-carousel": "^1.8.1"
   },
-  "scripts": {
-    "start": "react-scripts start",
-    "build": "react-scripts build",
-    "test": "react-scripts test",
-    "eject": "react-scripts eject"
+  "devDependencies": {
```
Comment (devjiwonchoi) [inline package.json:27]: moved to devDeps

**Review by jbeat30 — APPROVED** [review]
LGTM

**Review by jeonghoyang12 — APPROVED** [review]
LGTM

**Review by ggkim0614 — APPROVED** [review]
LFGTM!

**Pull request merged**